### DISCUSSION TITLE
Stability releases 🚀

### DISCUSSION THREAD

**Discussion by assafelovic** [assafelovic/gpt-researcher#12] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/assafelovic/gpt-researcher/releases/tag/v1.0.1'>Stability releases 🚀</a>.</em>

**Comment by luismccoy** [comment]
Is there a way to just force to use gpt 3.5 API instead of the default Gpt 4?

**Reply by ghost** [reply]
I would also like to ask the same question. GPT-4 is only available to customers having a record on payment for OpenAI. Until they open it up to all users, new customers for gptresearcher will not be able to use it.

**Reply by Godecule** [reply]
Same here ! Any ideas to put that off?

**Reply by ghost** [reply]
I have configured gpt3.5 turbo 16k 0613 as smartllm in the config file instead of the gpt-4. Now it works. This is the extract from the file:
 
$        self.smart_llm_model = os.getenv("SMART_LLM_MODEL", "gpt-3.5-turbo-16k-0613")
